docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
 
 There are several steps that must be performed in order to use OVE to control a display.
 
-## Setting up OVE
-
 Before using OVE, you will need to install [**OVE Core**](https://github.com/ove/ove), any [**OVE Apps**](https://github.com/ove/ove-apps), and any [**OVE Services**](https://github.com/ove/ove-services) that you intend to use. Installation guidelines can be found in the [OVE Installation Guide](./INSTALLATION.md).
 
-You will also need to modify the [``ove/packages/ove-core/src/client/Spaces.json``](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file to define a new ``space`` in OVE core.
+## Setting up OVE
 
-Note that all services need to hosted somewhere that is accessible from the computers connected to the monitors that will be used in the display.
```

Comment:
I think that it would still be useful to make a statement like this.

Reply:
The problem is that at the time that OVE is installed it may not be known how the machines that will be running the OVE clients will be connected to the network.

(As a real-world example of this, we deployed OVE to a server with the intention of displaying things on the GDO and DODev, but then tried to display things on MultiTaction displays that we didn't know we'd receive at the time we did the installation.)

This document describes how to "create" (instantiate?) an OVE space. It is at *this time*, not only the *installation time* that OVE was installed on a server, that you need to ensure that the server and client browsers can access each other.

I agree that the previous phrasing was imperfect. How about:

> You must ensure that the OVE core server, and all OVE services, are accessible from the computers connected to the monitors that will be used in the display.